### PR TITLE
Subscribe to multiple channels

### PR DESCRIPTION
**Pull request by bnolan**
.subscribe() accepts an array of channel names, messages broadcast to any subscribed channel will be received.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -54,15 +54,23 @@ module.exports = function (opts) {
...
-      var channel = get(name)
-      server.emit('subscribe', channel.name)
-      channel.subscribers.push(res)
-      eos(res, function () {
-        var i = channel.subscribers.indexOf(res)
-        if (i > -1) channel.subscribers.splice(i, 1)
-        if (!channel.subscribers.length && channel === channels[channel.name]) delete channels[channel.name]
+
+      var app = name.split('/')[0]
+      var channelNames = name.split('/')[1]
```
Comment (mafintosh) [inline server.js]: i think you should use `channelNames = name.slice(app.length + 1)` so channels still can contain `/` in their name

**Comment by bnolan** [comment]
Updated the PR with a test for slashes in names. Do you want me to rebase?

**Comment by bnolan** [comment]
Heya @mafintosh do you need anything from me here?

**Comment by bnolan** [comment]
Bump? :'(

**Comment by mafintosh** [comment]
@bnolan sorry about that - could you rebase against master? then I'll merge it :)

**Comment by bnolan** [comment]
I think that's it! :)

**Comment by mafintosh** [comment]
Nice. This isn't a breaking change right? (assuming you only use a single channel ofcourse)

**Comment by bnolan** [comment]
Nope, not breaking, works exactly the same for single channels.

On Wed, May 27, 2015 at 8:47 AM, Mathias Buus <notifications@github.com>
wrote:

> Nice. This isn't a breaking change right? (assuming you only use a single
> channel ofcourse)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mafintosh/signalhub/pull/3#issuecomment-105660075>.
>



-- 
Regards,
Ben Nolan

http://bennolan.com/

skype: bennolan
cell: +64 21 770 662

**Comment by mafintosh** [comment]
Nice. I'll release this as soon as i get to my computer

**Comment by mafintosh** [comment]
Out in 4.1.0. Thanks!

**Comment by bnolan** [comment]
Thank you! Yay. :)

On Wednesday, May 27, 2015, Mathias Buus <notifications@github.com> wrote:

> Out in 4.1.0. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mafintosh/signalhub/pull/3#issuecomment-105825272>.
>


-- 
Regards,
Ben Nolan

http://bennolan.com/

skype: bennolan
cell: +64 21 770 662

**Pull request merged**